qutebrowser/mainwindow/tabbedbrowser.py

```diff
@@ -217,19 +217,20 @@ def shutdown(self):
         for tab in self.widgets():
             self._remove_tab(tab)
 
-    def close_tab(self, tab):
+    def close_tab(self, tab, *add_undo=True):
```

Comment:
Sorry for being unclear - this should be `tab, *, add_undo=True)` - otherwise all positional arguments would land (as a list) in `add_undo` and break things.